pkg/adapter/adapter.go

```diff
@@ -127,6 +136,10 @@ func (adp *LocalAdapter) Dispatch(ctx context.Context, ref string, sync bool) er
 }
 
 func (adp *LocalAdapter) CheckHealth(ctx context.Context) error {
-	// TODO：return the status of boltdb
+	size, err := adp.content.Size()
+	if err != nil {
+		return err
+	}
+	logrus.Infof("local cache size: %s", humanize.Bytes(size))
```

Comment:
It's not necessary to print the message, and I think we just only check db is live on `CheckHealth`.